tests/system/small/engines/test_aggregation.py

```diff
@@ -25,6 +25,25 @@
 REFERENCE_ENGINE = polars_executor.PolarsExecutor()
 
 
+def apply_agg_to_all_valid(
+    array: array_value.ArrayValue, op: agg_ops.UnaryAggregateOp, excluded_cols=[]
+) -> array_value.ArrayValue:
```

Comment:
I can make some assumptions as to this helper function's purpose from the name, but a docstring would be lovely to clarify.